<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorDialog</class>
 <widget class="QDialog" name="GeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FRCGen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="fileNameLayout">
     <item>
      <widget class="QLabel" name="fileNameLabel">
       <property name="text">
        <string>File Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fileNameButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="languageLayout">
     <item>
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="languageCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="projectDirLayout">
     <item>
      <widget class="QLabel" name="projectDirLabel">
       <property name="text">
        <string>Project Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectDirEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="projectDirButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="buildCheck">
     <property name="text">
      <string>Build?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="deployLayout">
     <item>
      <widget class="QCheckBox" name="deployCheck">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Deploy?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="deploySpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="teamNumLabel">
       <property name="text">
        <string>Team #</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="teamNumEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="goButton">
     <property name="text">
      <string>Go!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
